lighthouse-cli/bin.js

```diff
@@ -77,6 +78,18 @@ async function begin() {
     configJson = require(`../lighthouse-core/config/${cliFlags.preset}-config.js`);
   }
 
+  if (cliFlags.budgetsPath) {
+    cliFlags.budgetsPath = path.resolve(process.cwd(), cliFlags.budgetsPath);
+    try {
+      const budgetsJsonStr = fs.readFileSync(cliFlags.budgetsPath, 'utf8');
+      /** @type {LH.Budgets.Json} */
+      const budgets = new Budgets(JSON.parse(budgetsJsonStr));
```

Comment:
we don't have any existing settings that are exactly analogous, but it feels like we should be loading the file here, as it does above, but the `new Budgets` initialization part should be happening in `config.json`?